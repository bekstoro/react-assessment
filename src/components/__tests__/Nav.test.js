import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';

import Nav from '@components/Nav';

describe('Nav', () => {
  it('Nav renders correctly', () => {
    const { container } = render(<Router><Nav /></Router>);
    expect(container).toMatchSnapshot();
  });
});
