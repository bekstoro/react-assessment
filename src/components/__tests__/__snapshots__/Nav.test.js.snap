// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Nav renders correctly 1`] = `
<div>
  <nav
    class="Nav__NavWrapper-sc-13du1es-0 cZYmJv"
  >
    <a
      aria-current="page"
      class="Nav__NavButton-sc-13du1es-1 idRylV active"
      href="/"
    >
      Task One
    </a>
    <a
      class="Nav__NavButton-sc-13du1es-1 idRylV"
      href="/task-two"
    >
      Task Two
    </a>
  </nav>
</div>
`;
